crates/move-stackless-bytecode/src/spec_hierarchy.rs

```diff
@@ -0,0 +1,195 @@
+use crate::function_target_pipeline::FunctionTargetsHolder;
+use move_model::model::{FunId, FunctionEnv, GlobalEnv, QualifiedId};
+use num::BigUint;
+use std::collections::BTreeSet;
+use std::fs;
+use std::path::Path;
+
+struct CallInfo {
+    display_name: String,
+    spec_id: Option<QualifiedId<FunId>>,
+}
+
+pub fn display_spec_hierarchy(env: &GlobalEnv, targets: &FunctionTargetsHolder, output_dir: &Path) {
+    let excluded_addresses = get_excluded_addresses();
+
+    for spec_id in targets.specs() {
+        let spec_env = env.get_function(*spec_id);
+        let spec_name = spec_env.get_name_str();
+
+        if let Some(fun_id) = targets.get_fun_by_spec(spec_id) {
+            let func_env = env.get_function(*fun_id);
+            let header = func_env.get_full_name_str();
+            write_spec_log_file(
+                env,
+                targets,
+                &func_env,
+                spec_name,
+                &header,
+                output_dir,
+                &excluded_addresses,
+            );
+        } else if targets.is_scenario_spec(spec_id) {
+            let header = format!("{} [scenario]", spec_env.get_full_name_str());
+            write_spec_log_file(
+                env,
+                targets,
+                &spec_env,
+                spec_name,
+                &header,
+                output_dir,
+                &excluded_addresses,
+            );
+        }
+    }
+}
+
+fn get_excluded_addresses() -> [BigUint; 5] {
+    [
+        0u16.into(),      // System address (core framework)
+        1u16.into(),      // Tests address
+        2u16.into(),      // Event address
+        3u16.into(),      // Stdlib address
```

Comment:
```
MoveStdlib -> 0x1
Sui -> 0x2
SuiSystem -> 0x3
```
`0` is not a valid address for a deployed package, but it can be the address of a local package, and it should not be excluded.
Let's not exclude the root package regardless of it's address.
Let's propagate this back to the stats pull request.